core/src/test/scala/unit/kafka/utils/SchedulerTest.scala

```diff
@@ -156,21 +167,23 @@ class SchedulerTest {
   /**
    * Verify that scheduler lock is not held when invoking task method, allowing new tasks to be scheduled
    * when another is being executed. This is required to avoid deadlocks when:
-   *   a) Thread1 executes a task which attempts to acquire LockA
-   *   b) Thread2 holding LockA attempts to schedule a new task
+   * a) Thread1 executes a task which attempts to acquire LockA
+   * b) Thread2 holding LockA attempts to schedule a new task
```

Comment:
I think the original indent is more readable

Reply:
> I think the original indent is more readable

Sorry，Idea open "Reformat code".
it has been rolled back